tests/test_core.py

```diff
@@ -8290,9 +8291,9 @@ def setUp(self):
 strict = make_run('strict', emcc_args=[], settings={'STRICT': 1})
 
 lsan = make_run('lsan', emcc_args=['-fsanitize=leak', '-O2'], settings={'ALLOW_MEMORY_GROWTH': 1})
-asan = make_run('asan', emcc_args=['-fsanitize=address', '-O2'], settings={'ALLOW_MEMORY_GROWTH': 1, 'INITIAL_MEMORY': '300mb'})
+asan = make_run('asan', emcc_args=['-fsanitize=address', '-g'], settings={'ALLOW_MEMORY_GROWTH': 1, 'INITIAL_MEMORY': '500mb'})
```

Comment:
is the change from `-O2` to `-g` intended? if so, why, the former seems better to me (but not for any great reason)

Reply:
No, that was a mistake reverting.